plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java

```diff
@@ -38,6 +38,7 @@
     public static final String TOPN_PUSHDOWN_ENABLED = "topn_pushdown_enabled";
     public static final String DOMAIN_COMPACTION_THRESHOLD = "domain_compaction_threshold";
     public static final String INSERT_BATCH_SIZE = "insert_batch_size";
+    public static final String NON_TRANSACTIONAL_INSERT = "non_transactional_insert";
```

Comment:
Can we make property/config name more self descriptive.
Maybe negate it and name `use_temporary_table_on_insert`?

Reply:
This is a unsafe feature so something which conveys that seems useful to me.

Reply:
Please rollback the change to `NON_TRANSACTIONAL_INSERT`